workflows/cloudify_system_workflows/snapshots/utils.py

```diff
@@ -424,7 +419,6 @@ def stage_db_schema_get_current_revision():
     if version['edition'] != 'premium':
         return None
     output = subprocess.check_output([
-        'sudo', '-u', snapshot_constants.STAGE_USER,
```

Comment:
I think these sudos are still needed, otherwise you don't haver permissions to access the migration.js, (and everything it accesses) as cfyuser (which the snapshot restore runs as).

Reply:
Oh, that's very strange, I don't recall removing that line. Reverting.